src/team/team.repository.ts

```diff
@@ -4,7 +4,8 @@ import { TeamModel } from './team.model';
 import { DuplicateUnique } from '../utils/error';
 import { ITeam } from './team.interface';
 
-export class TeamRepository {
+export class 
+TeamRepository {
```

Comment:
Why the new line? I think it is better in the original way:
```suggestion
export class TeamRepository {
```